core/src/main/java/org/ehcache/core/Ehcache.java

```diff
@@ -1079,4 +1079,25 @@ public AtomicInteger getActualRemoveCount() {
     }
   }
 
+  @Override
+  public String toString() {
+    return "statusTransitioner:\n    "
```

Comment:
I think `statusTransitioner.toString()` is wrong: what we need i something like:

`"status: " + getStatus()`